### PR TITLE
Prepare helm values for configuration management

### PR DESCRIPTION
**Pull request by kubasobon**
Towards https://github.com/giantswarm/giantswarm/issues/17386

### CODE REVIEW

**Comment by kubasobon** [comment]
Tested by comparing chart diffs on all installations

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@2.11.0
+  architect: giantswarm/architect@3.0.0
```
Comment (ubergesundheit) [inline .circleci/config.yml:3]: latest is `4.0.0`

**Comment by ubergesundheit** [comment]
Can you post a link to the configuration in `config` (or where this will live)?

**Comment by kubasobon** [comment]
https://github.com/giantswarm/config/pull/159

**Review by ubergesundheit — APPROVED** [review]
LGTM if it works for you

**Comment by ubergesundheit** [comment]
I can fix the build errors if you want

**Comment by kubasobon** [comment]
I haven't noticed those initially. That would be super cool @ubergesundheit!

**Comment by ubergesundheit** [comment]
> I haven't noticed those initially. That would be super cool @ubergesundheit!

Yeah, somewhere between `3.0.0` and `4.0.0`, `app-build-suite` got upgraded in `architect-orb`, for now I only silenced `kube-linter`.

It works now.

**Comment by ubergesundheit** [comment]
Regarding this switch to managed config, is this construct of having a special `image.registry` value obsolete now?

**Pull request merged**